js/main.js

```diff
@@ -54,11 +56,66 @@ function createElement(obj) {
   var photoDiv = document.createElement('div');
   photoDiv.style.backgroundImage = 'url(' + obj.recipe.image + ')';
   photoDiv.setAttribute('class', 'results-photo-div');
+  photoDiv.setAttribute('id', obj.recipe.label);
   resultHolder.appendChild(photoDiv);
   var dishName = document.createElement('h1');
+  dishName.setAttribute('class', 'dish-name');
   dishName.innerHTML = obj.recipe.label;
   resultHolder.appendChild(dishName);
 
   resultsContainer.appendChild(resultHolder);
```

Comment:
You shouldn't be appending a dynamically created DOM tree directly to the DOM, you should return that DOM tree from the function, and append it where you want from where the function is called. I would review `dom-creation` for a reminder on how to properly generate DOM trees for a set of data.